ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto

```diff
@@ -23,6 +23,11 @@ message Completion {
   // Identifies the exact type of the error.
   // For example, malformed or double spend transactions will result in a ``INVALID_ARGUMENT`` status.
   // Transactions with invalid time time windows (which may be valid at a later date) will result in an ``ABORTED`` error.
+  //
```

Comment:
This comment is slightly confusing. I'd expect it to talk about both success and error.

Reply:
Agreed. We should find a single place for documenting errors. Maybe `error.proto`?